src/main/java/com/contentgrid/gateway/ServiceDiscoveryProperties.java

```diff
@@ -6,6 +6,8 @@
 @Data
 @ConfigurationProperties(prefix = "servicediscovery")
 public class ServiceDiscoveryProperties {
+
     private boolean enabled = false;
     private String namespace = "default";
+    private long resync = 60;
```

Comment:
That seems fast? We'll always get a bunch of update events that we need to handle every 60s. The `Object.equals` will handle it early, but still...

Reply:
it's worse, I forgot the `* 1000L` part ;-)

the minimum resync interval is 1 sec - which runs fine locally pointing to a real K8s cluster
Why would once every minute not be fine in production ? 
(spring cloud k8s uses 30 secs by default)